core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -78,121 +79,225 @@
 
     private static final Logger logger = LoggerFactory.getLogger(WorkflowExecutor.class);
 
-    private MetadataDAO metadataDAO;
+    private final MetadataDAO metadataDAO;
 
-    private ExecutionDAO executionDAO;
+    private final ExecutionDAO executionDAO;
 
-    private QueueDAO queueDAO;
+    private final QueueDAO queueDAO;
 
-    private DeciderService deciderService;
+    private final DeciderService deciderService;
 
-    private Configuration config;
+    private final Configuration config;
+
+    private final MetadataMapperService metadataMapperService;
 
     private RateLimitingService rateLimitingService;
 
-    private ParametersUtils parametersUtils = new ParametersUtils();
+    private final ParametersUtils parametersUtils = new ParametersUtils();
+
+    private int activeWorkerLastPollInSecs;
```

Comment:
add javadoc to explain what this is

Reply:
That code was already there. No additions. I just grouped it with the rest of the final attributes